README.rst

```diff
@@ -85,12 +85,17 @@ Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 
 Changes
 =======
-0.4.3
+v0.4.4
+-------
+
+* Added force(check all po files) argument to validate command.
```

Comment:
Nit: Add a space between 'force' and '('

Reply:
I think we should add a little more substance to this, clarifying what the force argument does.